bpmsuite/bpmsuite70-businesscentral-monitoring-with-smartrouter.json

```diff
@@ -57,20 +57,48 @@
             "displayName": "KIE Admin Password",
             "description": "KIE administrator password",
             "name": "KIE_ADMIN_PWD",
-            "from": "[a-zA-Z]{6}[0-9]{1}!",
-            "generate": "expression",
+            "value": "adminPassword",
```

Comment:
This is set in order to match the user/pass set on the exec. server credentials to access the maven repo